Libraries/PushNotificationIOS/PushNotificationIOS.js

```diff
@@ -99,8 +108,13 @@ class PushNotificationIOS {
    * - `soundName` : The sound played when the notification is fired (optional).
    * - `category`  : The category of this notification, required for actionable notifications (optional).
    * - `userInfo` : An optional object containing additional notification data.
+   * - `repeatInterval` : The interval to repeat as a string.  Possible values: `minute`, `hour`, `day`, `week`, `month`, `year`.
    */
   static scheduleLocalNotification(details: Object) {
+    if (details) {
+      details.repeatInterval = REPEAT_INTERVAL[details.repeatInterval] || 0;
```

Comment:
Red screen with `JSON value '<null>' of type NSNull cannot be converted to NSNumber`. My thinking was it's going to be converted to `0`. Then, we have to add in a `details[@"repeatInterval"] || @"0"` and that should work.